lib/logstash/plugin_mixins/kafka_support.rb

```diff
@@ -0,0 +1,29 @@
+module LogStash module PluginMixins module KafkaSupport
+
+  def set_trustore_keystore_config(props)
+    props.put("ssl.truststore.type", ssl_truststore_type) unless ssl_truststore_type.nil?
```

Comment:
prior to #8 output had a [nil check](https://github.com/logstash-plugins/logstash-integration-kafka/commit/e6f7d3a8be6a606fb498a20ad1d21d48546689ac#diff-3ce6e7cf50af34dab48324fd49c5fb514373531921e89015f1974891f5f5ef87R244-R246) 
```ruby
  if ssl_truststore_location.nil?
    raise LogStash::ConfigurationError, "ssl_truststore_location must be set when SSL is enabled"
  end
```
since Kerberos authentication was introduced at: https://github.com/logstash-plugins/logstash-integration-kafka/commit/e6f7d3a8be6a606fb498a20ad1d21d48546689ac#diff-3ce6e7cf50af34dab48324fd49c5fb514373531921e89015f1974891f5f5ef87R244-R246

input did not have such a check and truststore is optional according to the docs regardless of consumer/producer.
(despite the 'revert' this should still address the problematic case presented in https://github.com/logstash-plugins/logstash-integration-kafka/pull/8)

https://github.com/logstash-plugins/logstash-integration-kafka/pull/60/files#diff-886ed388751b8abe78d71e1bbb13450fd499ef48af6d95f06795f6aa8d825398R53